src/app/components/Shared/input.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export default function Input({name, id, placeholder}) {
```

Comment:
Same as above… Why nest it in a Shared folder. From my understanding src/app/components is for shared components. So no need to nest it even deeper.